packages/@webex/plugin-meetings/src/multistream/remoteMedia.ts

```diff
@@ -102,6 +102,29 @@ export class RemoteMedia extends EventsScope {
     this.id = `RM${remoteMediaCounter}-${this.receiveSlot.id}`;
   }
 
+  /**
+   * Supply the width and height of the video element
+   * to restrict the requested resolution to this size
+   * @param width width of the video element
+   * @param height height of the video element
+   */
+  public setSizeHint(width, height) {
+    // only base on height for now
+    let fs: number;
+
+    if (height < 135) {
+      fs = 60;
+    } else if (height < 270) {
+      fs = 225;
```

Comment:
let's use same values we have in getMaxFs(), so 240 here

Reply:
changed to use 240